src/components/LightBoxes/ForkMap.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+
+class ForkMap extends Component {
```

Comment:
i don't want to make too much extra work, but this should really be split into smaller subcomponents e.g. permIcon, clearFloat, and maybe input group components

Reply:
Hmm, I could see about doing that.